test/clj/puppetlabs/trapperkeeper/testutils/webserver.clj

```diff
@@ -55,3 +58,25 @@
      ~port-var
      {}
      ~@body))
+
+(defn get-jetty-mbean-object-names
+  "TODO SPLAIN"
+  []
+  (let [mbeanserver (ManagementFactory/getPlatformMBeanServer)]
+    (.queryNames mbeanserver
+                 nil
+                 (ObjectName. "org.eclipse.jetty.jmx:type=mbeancontainer,*"))))
```

Comment:
I noticed that in https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/master/test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj#L116 we're using `clojure.java.jmx/mbean-names`, which looks like it does almost exactly this. Would it make sense to use that here too?